File: tests/fixtures/codegen/expected/typescript/runtime/queries/events/RenameEvent.ts
import type { RenameEventParams } from "../../../types/queries/events/RenameEvent.js";
export type { RenameEventParams } from "../../../types/queries/events/RenameEvent.js";
import type * as pgsid from "./pgsid/queryable.js";
export const renameEventSql = "UPDATE public.events SET note = $1 WHERE id = $2;";
export async function renameEvent(db: pgsid.Queryable, params: RenameEventParams): Promise<void> {
    await db.query(renameEventSql, [params["note"], params["id"]]);
}

File: tests/fixtures/codegen/expected/go/queries/events/renameevent.go
package events

import "context"

const RenameEventSQL = "UPDATE public.events SET note = $1 WHERE id = $2;"

type RenameEventParams struct {
	Note *string `db:"note"`
	Id   *int64  `db:"id"`
}

func (q *Queries) RenameEvent(ctx context.Context, params RenameEventParams) error {
	_, err := q.db.Exec(ctx, RenameEventSQL, params.Note, params.Id)
	return err
}
